src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1481,21 +1484,20 @@ because xbuild doesn't support framework reference assemblies.
 </Target>
   
 <Target Name="_CreateNativeLibraryArchive"
-        Condition="@(EmbeddedNativeLibrary) != ''" 
+        Condition="'$(AndroidApplication)' != 'True' And @(EmbeddedNativeLibrary) != ''" 
```

Comment:
Doesn't this doesn't need to be `'@(EmbeddedNativeLibrary)'` (quotes around `@(EmbeddedNativeLibrary)`)?